build.assets/Makefile

```diff
@@ -196,13 +203,35 @@ buildbox-arm-fips: buildbox-fips
 		--cache-from $(BUILDBOX_ARM_FIPS) \
 		--tag $(BUILDBOX_ARM_FIPS) -f Dockerfile-arm-fips .
 
+#
+# Builds a Docker buildbox which has tools needed to install grpc-tools npm package on arm64.
+# See the `grpc-teleterm` target in the main Makefile for more details.
+#
+.PHONY:buildbox-teleterm
+buildbox-teleterm: buildbox
+	@if [[ $${DRONE} == "true" ]] && ! docker inspect --type=image $(BUILDBOX_TELETERM) 2>&1 >/dev/null; then docker pull $(BUILDBOX_TELETERM) || true; fi;
```

Comment:
Some of the other buildbox targets have an additional conditional:

https://github.com/gravitational/teleport/blob/a220e2d77d84d9fffc9f2691c3672108c988b21c/build.assets/Makefile#L119-L120

I'm not 100% sure if buildbox-teleterm needs one as well, but since for now it's going to be executed only on our dev machines, I think I can skip it.